components/IconCard.js

```diff
@@ -24,7 +24,7 @@ export default function IconCard({ iconName }) {
       </button>
       <div
         aria-live="assertive"
-        className="pointer-events-none fixed inset-0 flex items-end px-4 py-6 sm:items-start sm:p-6"
+        className="pointer-events-none fixed inset-0 z-40 flex items-end px-4 py-6 sm:items-start sm:p-6"
```

Comment:
Wouldn't now the text overlap the navbar?

Reply:
No it's a alert it goes away after 1.5sec

Reply:
Actually there was no issue, but in recent code changes, I think someone gave navbar zindex of 30 to solve another issue, which caused the alert component display under the navbar, that's my understanding of the current issue.